src/build.sh

```diff
@@ -89,12 +89,18 @@ function kernel_build()
   fi
 
   if [ -x "$(command -v nproc)" ]; then
-    parallel_cores=$(nproc --all)
+    parallel_cores="-j$(nproc --all)"
   else
-    parallel_cores=$(grep -c ^processor /proc/cpuinfo)
+    parallel_cores="-j$(grep -c ^processor /proc/cpuinfo)"
   fi
 
-  command="make -j$parallel_cores ARCH=$arch $cross_compile"
+  if [[ -n "${options_values['OPTIMIZE']}" ]]; then
+    optimizations="CC=\"ccache gcc -fdiagnostics-color\" $parallel_cores"
+  else
+    optimizations="$parallel_cores"
```

Comment:
I think we should also allow users to say how many percent of their CPU they want to utilize, for example, something like `kw build --cpu-utilization "50%"`. I want full-speed compilation most of the time, but sometimes I want to keep some CPU resources for other tasks.

Reply:
But do you think a percentage would work alright? How do you propose we make those calculations?